src/widgets/table.rs

```diff
@@ -5,84 +5,213 @@ use strum::{Display, EnumString};
 use unicode_width::UnicodeWidthStr;
```

Comment:
Add `#![warn(missing_docs)]` to avoid regression in this file?

Reply:
good idea